docs/pages/upgrading/self-hosted-automatic-agent-updates.mdx

```diff
@@ -23,216 +23,68 @@ present in their package repositories. As Teleport 14 won't be published to
 the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).
 Alternatively, you can use the `stable/rolling` channel, which contains
 Teleport v13.3.2 forward, including future major releases.
-
 </Admonition>
 
 ## Prerequisites
 
 - Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
   which describes the sequence in which to upgrade components of your cluster.
-- Self-hosted Teleport cluster v13.0 or higher.
+- Self-hosted Teleport cluster running at least v14.3.7 or v15.1.3. 
 - (!docs/pages/includes/tctl-tsh-prerequisite.mdx!)
 - (!docs/pages/includes/tctl.mdx!)
 
-The version server must be hosted on a webserver with trusted TLS certificates
-and reachable by all agents. You must have either:
-  - A public Amazon S3 or Google Cloud Storage bucket.
-  - A web server accessible from all agents with valid TLS certificates.
-
-If you do not have an S3 or GCS bucket for your version server, this guide
-provides an example of how to create one using Terraform. You can [install
-Terraform](https://developer.hashicorp.com/terraform/downloads) to spin up a
-demo version server along with this guide so you can get started with automatic
-agent upgrades. Terraform is not required to set up a version server. 
+<Admonition type="note">
```

Comment:
I think this admonition would make more sense in the changelog. Users reading this guide won't see instructions to set up a public web server. We could also provide a brief architectural overview in the introduction of this page that indicates that the Proxy Service hosts the version server. This way, we don't risk adding noise for users with an admonition.

Reply:
I removed the note, we already have a changelog entry for this